parsers/fastlyVectorParser.js

```diff
@@ -24,6 +25,7 @@ module.exports = function parse(line) {
     totalTime: totalTime,
     firstByteTime: Number(firstByteSecs) * 1000,
     api: 'vector-tiles',
-    origin: 'fastly'
+    origin: 'fastly',
+    server: server
```

Comment:
should have fsv.escapeField(server, 100) just to make sure it matches the schema... obviously its extremely unlikely that it would ever matter :)